docs/index.rst

```diff
@@ -1,14 +1,162 @@
-Welcome to Betelgeuse's documentation!
-======================================
+Betelgeuse documentation
+========================
 
-Betelgeuse reads standard Python test cases and offers tools to interact with
-Polarion. Possible interactions are:
+.. contents:: Topics
 
-* Automatic creation and update of Requirements and Test Cases from a Python
+.. _what_is_betelgeuse:
+
+What is Betelgeuse?
+```````````````````
+
+Betelgeuse is a python program that reads standard Python test cases and offers
+tools to interact with Polarion. Possible interactions are:
+
+* Automatic creation/updation of Requirements and Test Cases from a Python
   project code base.
 * Creation of Test Runs based on a jUnit XML file.
 
-Documentation contents:
+Betelgeuse uses `testimony <https://pypi.python.org/pypi/testimony>`_ and
+pylarion projects to parse test cases and communicate with polarion
+respectively.
+
+.. _prerequisites:
+
+Prerequisites
+`````````````
+Login to polarion in a browser to check if the login user has permissions to do
+the following:
+
+* add/update test case
+* add/update requirements
+* create test runs
+
+.. _quick_start:
+
+Quick Start
+```````````
+1. betelgeuse uses pylarion to interact with polarion. Install pylarion from its
+   source.
+
+   .. note::
+
+     - It may be possible that the latest version of betelgeuse may not work
+       correctly with some versions of pylarion.  In this case, please use an
+       alternate working version of pylarion.
+     - Read pylarion documentation and set up .pylarion config file as
+       required.
```

Comment:
I respectfully disagree. I like to keep betelgeuse project as atomic as possible.  This doc already says above that the user should read pylarion documentation to set .pylarion config file.  Any sample .pylarion file should live in the pylarion project README in my opinion.

Also pylarion documentation clearly says that the config file can live in current directory, user's home directory, or /etc/pylarion/pylarion.cfg.  I don't think we need to restrict betelgeuse users to keep the config in home directory only.